apps/certs/admin.py

```
@@ -34,11 +53,18 @@ class CertbotCertificateAdmin(CertificateProvisioningMixin, admin.ModelAdmin):
+        "auto_renew",
         "last_requested_at",
     )
     search_fields = ("name", "domain", "email")
     readonly_fields = ("last_requested_at", "last_message")
-    actions = ["generate_certificates", "request_certbot", "verify_certificates"]
+    actions = [
+        "generate_certificates",
+        "request_certbot",
+        "verify_certificates",
+        "renew_due_certificates",
+    ]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)